OptimizelySDK/Optimizely.cs

```diff
@@ -62,6 +68,7 @@ public class Optimizely
             ErrorHandler = errorHandler ?? new NoOpErrorHandler();
             Bucketer = new Bucketer(Logger);
```

Comment:
Get rid of bucketer from this class because all bucketing will be done in the decision service